cli/src/config/templates.toml

```diff
@@ -281,9 +288,39 @@ separate(" ",
         if(commit.git_head(), label("git_head", "git_head()")),
         format_short_commit_id(commit.commit_id()),
         if(commit.conflict(), label("conflict", "conflict")),
+        if(config("ui.show-cryptographic-signatures").as_boolean(),
+          format_short_cryptographic_signature(commit.signature())),
       )
 '''
 
+# signature verification is slow, disable by default
+'format_detailed_cryptographic_signature(signature)' = '''
+if(signature,
+  separate(" ",
+    format_cryptographic_signature_status(signature),
+    label("signature display", signature.display()),
+    label("signature key", signature.key()),
```

Comment:
nit: these `label()`s are redundant. `signature.display()` is transformed to `label("signature display", signature_display_value)`.